docs/reference/commandline/run.md

```diff
@@ -372,16 +372,18 @@ Under these conditions, you can pass any size less than the backing filesystem s
 For the `windowsfilter`, `btrfs`, and `zfs` storage drivers, you cannot pass a
 size less than the Default BaseFS Size.
 
-
 ### <a name="tmpfs"></a> Mount tmpfs (--tmpfs)
 
+The `--tmpfs` flag lets you create a `tmpfs` mount.
+
+The options that you can pass to `--tmpfs` are identical to the Linux `mount -t
+tmpfs -o` command. The following example mounts an empty `tmpfs` into the
+container with the `rw`, `noexec`, `nosuid`, `size=65536k` options.
+
```

Comment:
For a follow-up, we can consider cross-linking this to `--mount type=tmpfs` https://docs.docker.com/storage/tmpfs/#tmpfs-mounts

Reply:
carried in #4712